uw-frame-components/css/themes/uw-madison-variables.less

```diff
@@ -1,9 +1,9 @@
 /* MyUW-Madison colors */
-@color1: #b70101;         // Badger Red
+@color1: #c5050c;         // Badger Red
 @color2: #990000;         // Medium Red
 @color3: #660000;         // Dark Red
 @color4: #F7F5E8;         // Cream
-@link-color: #660000;     // Dark Red
+@link-color: #0479a8;     // wisc.edu blue
```

Comment:
That link color is actually the base color that the accent pallette is based upon, so there's no need to make any changes to the theme. It's just a matter of using the "md-accent" class on button links to ensure consistency with normal <a> tags.